### PR TITLE
Add Teams Notification feature

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do? What is the motivation?

- Add configuration instructions to setup notification channels to Teams
- Audit Teams instructions with the UI
- Add reference link from Monitors docs

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/account_management/teams.md [hunk 1]
```diff
@@ -76,14 +68,23 @@ Team handles that aren't associated with a defined team in Datadog behave simila
...
-- [Service Level Objectives][8]
+- [Dashboards][4]
+- [Incidents][5]
+- [Monitors][6]
+- [Resource Catalog][7]
+- [Service Catalog][8]
+- [Service Level Objectives][9]
 - Synthetic Tests, Global Variables, Private Locations
 
+### Send notifications to a separate communication channel
```
Comment (SimoneT-DD) [inline content/en/account_management/teams.md:79]: Hey @estherk15,
I'm not sure "separate" is the right word here? Maybe "specific" would work better? WDYT?

**Review by SimoneT-DD — APPROVED** [review]
Hi @estherk15, 

this looks good to me, I only have one minor comment

**Review by buraizu — APPROVED** [review]
Looks good overall and approving, but I agree that `specific` might be a better word as mentioned [here](https://github.com/DataDog/documentation/pull/22328#pullrequestreview-1958036787)

File: content/en/account_management/teams.md (hunk 1 above)
Comment (estherk15) [reply content/en/account_management/teams.md:79]:
```suggestion
### Send notifications to a specific communication channel 
```

File: content/en/monitors/notify/_index.md [hunk 2]
```diff
@@ -206,4 +206,4 @@ Message variables auto-populate with a randomly selected group based on the scop
 [14]: /monitors/guide/how-to-set-up-rbac-for-monitors/
 [15]: /monitors/types
 [16]: /monitors/guide/recovery-thresholds/
-[17]: /account_management/teams/
+[17]: /account_management/teams/#send-notifications-to-a-separate-communication-channel
```
Comment (estherk15) [inline content/en/monitors/notify/_index.md:209]:
```suggestion
[17]: /account_management/teams/#send-notifications-to-a-specific-communication-channel
```

**Pull request merged**